jina/serve/runtimes/monitoring.py

```diff
@@ -17,4 +20,19 @@ def _setup_monitoring(self):
 
             from prometheus_client import start_http_server
 
-            start_http_server(self.args.port_monitoring, registry=self.metrics_registry)
+            try:
+                start_http_server(
+                    self.args.port_monitoring, registry=self.metrics_registry
+                )
+            except OSError as e:
+                if (
+                    e.args[0] == 98
```

Comment:
what is this 98? lets create some constants or something

Reply:
OSError: port already in use

Reply:
updated